crates/net/discv5/src/metrics.rs

```diff
@@ -48,6 +48,13 @@ pub struct DiscoveredPeersMetrics {
     /// Total number of sessions established by [`discv5::Discv5`], that pass configured
     /// [`filter`](crate::filter) rules.
     total_established_sessions_custom_filtered: Counter,
+    /// Total number of unverifiable ENRs discovered by [`discv5::Discv5`].
+    ///
+    /// These are peers that fail [`discv5::Discv5`] session establishment, because the UDP socket
+    /// they're making a connection from doesn't match the UDP socket advertised in their ENR.
+    /// These peers will be denied a session (and hence can't make it into kbuckets) until they
+    /// have update their ENR, to reflect their actual UDP socket.
+    total_unverifiable_enrs_raw: Counter,
```

Comment:
Let's adhere to the naming convention with the suffix `_total` (see https://github.com/paradigmxyz/reth/issues/8150)
```suggestion
    unverifiable_enrs_raw_total: Counter,
```

Also, what does "raw" mean here?

Reply:
raw is referring to "not filtered", "directly from underlying sigp/discv5 library". the raw metrics would probably make more sense in the sigp/discv5 repo, but for now, they can live here. open to suggestions for better name than "raw".